openedx/core/djangoapps/user_authn/api/tests/test_views.py

```diff
@@ -396,7 +397,8 @@ def test_welcome_page_context(self):
         """
         Test MFE Context API response for welcome page
         """
-        self.query_params.update({'is_welcome_page': True})
+        redirect_url = 'https://welcome.page.redirect.org'
+        self.query_params.update({'is_welcome_page': True, 'next': redirect_url})
```

Comment:
We should also verify welcomePageRedirectUrl

Reply:
Addressed!